cores/migrations/20240725012747_RenameTablesToPlural.up.sql

```diff
@@ -0,0 +1,13 @@
+-- This migration is NOT BLUE/GREEN SAFE
+-- Any 'old' code that tries to access to table after this migration runs will FAIL
+ALTER TABLE category
+RENAME TO categories;
+
+ALTER TABLE markdown
+RENAME TO markdowns;
+
+ALTER TABLE pagesnapshot
+RENAME TO page_snapshots;
+
+ALTER TABLE categorymarkdown
+RENAME TO markdown_categories;
```

Comment:
Make all the tables plural

And for somereason I liked markdown_categories or category_markdowns 🤷